modules/462-loki/openapi/doc-ru-config-values.yaml

```
@@ -40,49 +36,75 @@ properties:
       Структура, аналогичная `spec.tolerations` пода Kubernetes.
 
       Если ничего не указано или указано `false`, будет [использоваться автоматика](https://deckhouse.ru/documentation/v1/#выделение-узлов-под-определенный-вид-нагрузки).
-  resourcesRequests:
+  resourcesManagement:
     description: |
-      Настройки максимальных значений CPU и памяти, которые может запросить под при выборе узла.
-
-      Если модуль [vertical-pod-autoscaler](../302-vertical-pod-autoscaler/) выключен, максимальные значения становятся значениями по умолчанию.
+      Варианты управления ресурсами Loki.
```

```suggestion
      Варианты управления ресурсами Loki.
```
Все-таки это не варианты.  Варианты - это это выбор. Ну т.е возможно параметр mode, а тут родительский уровень - тут что-то типа настройки....  Ну т.е. это секция которая позволяет настроить управление ресурсами а не выбрать вариант управления ИМХО

---

Loki resources management options. Опции тут, значит, не выборка, а как настройка. Окей.

---

@MariaMakeeva настройка или настройки?